src/protocol.rs

```diff
@@ -1,3 +1,5 @@
+pub mod operation;
```

Comment:
@calebbourg Have you discovered how we can have this just be a privately shared module among the crate only?

Reply:
I added a new issue for this https://github.com/Jim-Hodapp-Coaching/esp32-wroom-rp/issues/30

I think it's worth creating a separate PR and thinking through as it might mean moving `Operation` out of the `protocol` namespace.